puppet/modules/leap/manifests/logfile.pp

```diff
@@ -0,0 +1,26 @@
+#
+# make syslog log to a particular file for a particular process.
+#
+
+define leap::logfile($process=$title) {
+  $logfile = "/var/log/leap/${title}.log"
+
+  rsyslog::snippet { "50-${name}":
```

Comment:
Why use ${name} here, but ${title} earlier? It seems weird to have $process, $title, $name all being used in different places.